core/iwasm/aot/aot_loader.c

```diff
@@ -2076,6 +2065,13 @@ load_relocation_section(const uint8 *buf, const uint8 *buf_end,
         goto fail;
     }
 
+    symbols = loader_malloc(sizeof(*symbols) * symbol_count, error_buf,
+                            error_buf_size);
+    if (symbols == NULL) {
+        goto fail;
+    }
+    memset(symbols, 0, sizeof(*symbols) * symbol_count);
```

Comment:
No need to memset again, loader_malloc already memset the buffer.

Reply:
done